pkg/schema/openapi.go

```diff
@@ -5,21 +5,56 @@ package schema
 
 import (
 	"fmt"
+	"sort"
 
 	"github.com/k14s/ytt/pkg/yamlmeta"
 )
 
 // keys used when generating an OpenAPI Document
 const (
+	titleProp              = "title"
 	typeProp               = "type"
-	defaultProp            = "default"
+	additionalPropsProp    = "additionalProperties"
+	formatProp             = "format"
 	nullableProp           = "nullable"
+	deprecatedProp         = "deprecated"
```

Comment:
There is another story in flight to add deprecated, so it's not yet tested even though it's here. I know there has been a lot of collaboration on this, but I think we should take this key out, and add it in the story for `@schema/deprecated`. 

For example, it is possible that this PR get accepted, and we have this key for `deprecated` here. Then pretend some new information comes to light and we no longer want to move forward with the `@schema/deprecated` feature at all. Then this code will never be used. While it is unlikely, I think it's good practice to wait.

Reply:
Makes sense, I will remove it.